tutorials/stats-sensor-space/75_cluster_ftest_spatiotemporal.py

```diff
@@ -324,16 +367,20 @@
     # clean up viz
     mne.viz.tight_layout(fig=fig)
     fig.subplots_adjust(bottom=.05)
+    plt.show()
 
 
 # %%
 # Exercises
 # ----------
 #
 # - What is the smallest p-value you can obtain, given the finite number of
-#   permutations?
-# - use an F distribution to compute the threshold by traditional significance
-#   levels. Hint: take a look at :obj:`scipy.stats.f`
+#   permutations? You can find the answers in the references
+#   :footcite:`MarisOostenveld2007` :footcite:`Sassenhagen2019`.
```

Comment:
this would be a good place to try out something I don't think we've tried before:

```suggestion
#   :footcite:p:`MarisOostenveld2007,Sassenhagen2019`.
```
(note the `:p:` --- in theory this should render as `Author (year)`)